invenio.cfg

```diff
@@ -419,14 +419,19 @@ RDM_CUSTOM_FIELDS = [
 
 # Custom fields UI components
 RDM_CUSTOM_FIELDS_UI = [
-    # journal
-    JOURNAL_CUSTOM_FIELDS_UI,
+    {
+        "section": _("Publishing information"),
```

Comment:
This can, and must, be challenged. It's also set for discussion during next week.